assets/html/idle-queue/index.html

```diff
@@ -0,0 +1,174 @@
+
+	<!-- Titles -->
+	<title>Idle Queue</title>
+	<meta name="apple-mobile-web-app-title" content="PWA Name">
+	<meta name="application-name" content="PWA Long Name">
+	<meta name="description" content="All the tools you need to build and deploy your Progressive Web Apps.">
+	<!-- Titles:end -->
+
+	<link rel="manifest" href="/assets/manifest.json">
+
+	<!-- Responsiveness -->
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Browser shell -->
+		<meta name="theme-color" content="#000000">
+
+	<!-- Add to home screen for iOS -->
+		<meta name="apple-mobile-web-app-title" content="PWA Name">
+		<meta name="apple-mobile-web-app-capable" content="yes">
+		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+		<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/icons/apple-touch-icon.png">
+		<link rel="apple-touch-icon" href="images/touch/apple-touch-icon.png">
+
+	<!-- Tile icons for Windows -->
+		<meta name="msapplication-config" content="/assets/images/icons/browserconfig.xml">
+		<meta name="msapplication-TileImage" content="/assets/images/icons/mstile-144x144.png">
+		<meta name="msapplication-TileColor" content="#eff4ff">
+
+	<!-- Favicons -->
+		<link rel="icon" type="image/png" sizes="32x32" href="/assets/images/icons/favicon-32x32.png">
+		<link rel="shortcut icon" href="/assets/images/icons/favicon.ico">
+
+	<!-- Safari pin icon -->
+		<link rel="mask-icon" href="/assets/images/icons/safari-pinned-tab.svg" color="#000000">
+
+	<!-- CSS -->
+		<link rel="stylesheet" href="../../dist/css/main.css">
+		<!-- Lazy Load non-critical stylesheets -->
+		<link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700|Source+Serif+Pro" rel="stylesheet" media="none" onload="if( media!='screen' ) media='screen' ">
+	<!-- CSS:end -->
+	<script>
+		window.xwpQueue = [];
+	</script>
+</head>
+<body>
+	<header class="l-header center">
+
+		<nav class="nav">
+			<p class="nav__logo">XWP scripts</p>
+			<ul class="nav__items">
+				<li class="nav__item">
+					<a class="nav__link" href="../">PWA</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__line" href="../lazy-load">Lazy Loading</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link is-active" href="#">Idle Queue</a>
+				</li>
+				<li class="nav__item">
+					<a class="nav__link" href="../lazy-embeds">Third Party Embeds</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+
+
+	<main class="center">
+		<article class="stack">
+			<h1>Idle Queue</h1>
+			<h2>Goal</h2>
+
+			<p>
+				We want to delay third party js. To achieve this goal, we use a solution based on <code>idle-until-urgent</code> idea developed by <a href="https://philipwalton.com/articles/idle-until-urgent/">Philip Walton</a> <br />
+			</p>
+			<h2>HTML required</h2>
+			<p>We want to create a queue for two different cases.</p>
+			<h3>Inline self-executable functions</h3>
+			<p>Let's looks at this Facebook function</p>
+			<xmp>
```

Comment:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/xmp), `xmp` is obsolete. How about switching to `pre` instead?

Reply:
`pre` + `code` are used instead of `xmp`